bbp/nse/bluepy/default.nix

```diff
@@ -13,20 +13,17 @@
 
 pythonPackages.buildPythonPackage rec {
     pname = "bluepy";
-    version = "0.12.5";
+    version = "0.12.6";
     name = "${pname}-${version}";
 
     src = fetchgitPrivate {
         url = config.bbp_git_ssh + "/nse/bluepy";
-        rev = "834babaf6778f9b7474e5b824dd98e0d8be90c16";
-        sha256 = "0kvkyhzlp29i3sr999h9i7jns4xxjfd9cz0121kib6mihvd2p3xv";
+        rev = "afc3101a18ee46c269da8a03dd257946af54d117";
+        sha256 = "0j3r7xcwkpn9mmm3fpxar4s3nd83h1qbv66d2aayf90sy4lf1hri";
     };
 
     # TODO: remove once `bluepy` dependencies are revised
     preConfigure = ''
-        # use >= deps and not absolute == to avoid conflicts....
```

Comment:
`shapely` was the last pinned dependency remaining; and it was unpinned in one of `0.12.6` commits